app/lib/search/query_builder.rb

```diff
@@ -106,7 +103,7 @@ def unfilterise(field = '')
     end
 
     def order_query
-      order_query_builder_class.new(
+      OrderQueryBuilder.new(
```

Comment:
Good stuff. Hangover from the Advanced Search stuff?

Reply:
think so. only place it is used so this looks clearer